core/commons/che-core-commons-lang/src/main/java/org/eclipse/che/commons/lang/execution/ParametersList.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.commons.lang.execution;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represent and configure program parameters
+ */
+public class ParametersList {
```

Comment:
Because java.lang.ProcessBuilder constructor requires list of the parameters and  it's easier to handle as List